pkgs/tools/typesetting/skribilo/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv, fetchurl, pkgconfig, gettext
+, guile, guile-reader, guile-lib
+, ploticus, imagemagick
+, ghostscript, transfig
+, enableEmacs? false, emacs? null
```

Comment:
I do not think we ever use the question mark (default value separator) without a space from both sides.

Reply:
Scheme muscle memory :)